build

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# Locally builds the docker container and runs tests.
+#
+# The test sets to run (gpu, cpu, tpu) can be specified as arguments.  By
+# default all tests are executed.
+
+PROJECT_ID="$(gcloud config get-value project)"
+if [ -z "${PROJECT_ID}" ]; then
+  echo "You must set a project ID with `gcloud config set`"
+  exit 1
+fi
+
+MODEL="gs://deepvariant/models/DeepVariant/0.7.2/DeepVariant-inception_v3-0.7.2+data-wgs_standard"
+IMAGE="gcr.io/${PROJECT_ID}/gcp-deepvariant-docker:HEAD"
+
+docker build -t "${IMAGE}" .
+docker push "${IMAGE}"
+
+for profile in ${@:-cpu gpu tpu}; do
+  echo "Running ${profile} tests"
+  docker run -v "${HOME}/.config:/root/.config" \
```

Comment:
Copy pasting from prv PR 

"
  nmousavi@
I would suggest using gcloud builds submit (pointed to the cloudbuild.yaml file), so we could have a manual build/test exactly similar to the automated one.

  @kemp-google
This requires setting permissions up in the cloud project for the builder. It's possible, but there is no particular advantage: the cloud build environment is the same as your local docker environment (or, at least, it should be).

This is also significantly faster than using cloud build.
"

Reply:
The benefit is in line with DRY principals, this script is basically replicating what is instructed in cloudbuild.yaml. For example, upon new model release, you need to update model var in both places. If you still thinks having it here outweighs what I mentioned, you may want to model var and anything else can be moved to run_and_verify.sh (so we have a single point of future change).

Reply:
- The model will be moving into the DeepVariant docker container in the near future so it will end up being removed anyway.
- There isn't an easy way to import variables into the cloudbuild.yaml, so I can't effectively share the constants anyway
- Another option is to have a shell script that parses cloudbuild.yaml, but I think the complexity outweighs the value (we aren't likely to add many new platforms so changes aren't too likely)

Reply:
Both cloudbuild.yaml and this script pass model value to `run_and_verify.sh`, so if you move it into there, there is no need to define in the two places.

Reply:
Ah, I understand what you were trying to say above now.  Yes, moving it into run_and_verify seems like a reasonable change.  I'll send a PR to do that and then rebase this change.